src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -95,6 +96,10 @@ def phylo_trees():
             "pathogen_genome_count": 0,  # TODO: do we still need this?,
             "tree_type": phylo_run.tree_type.value,
             "user": {},
+            "group": {
+                "id": phylo_run.group.id,
+                "name": phylo_run.group.name,
+            },
```

Comment:
Are we still calling the old phylo trees endpoint?

Reply:
Yup. We will have to pick back up the migration when we can

Reply:
Oh yeah... @mmmmmmaya the v2 endpoint returns this information already, but the information is formatted differently. Do you want to just use the new endpoint?